exercises/run-length-encoding/src/test/java/RunLengthEncodingTest.java

```diff
@@ -0,0 +1,79 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class RunLengthEncodingTest {
+    private RunLengthEncoding runLengthEncoding;
+
+    @Before
+    public void setUp() {
+        runLengthEncoding = new RunLengthEncoding();
+    }
+
+    @Test
+    public void encodeEmpty() {
+        Assert.assertEquals("", runLengthEncoding.encode(""));
+    }
+
```

Comment:
It should only be the first test which doesn't have an `@Ignore`. Youv'e left it off the first two (my comment was meant for the second test, sorry if that wasn't clear) :)